### PR TITLE
raimo.Smallpdf version 2.3.0.0

### PR DESCRIPTION
**Pull request by Gashans**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22132)

### CODE REVIEW

**Comment by ChuckFerring** [comment]
It appears that this is a 64bit application but the specified installer is x86.

File: manifests/r/raimo/Smallpdf/2.3.0.0/raimo.Smallpdf.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: raimo.Smallpdf
+PackageVersion: 2.3.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (ItzLevvie) [inline manifests/r/raimo/Smallpdf/2.3.0.0/raimo.Smallpdf.installer.yaml:8]:
```suggestion
  Architecture: x64
```

File: manifests/r/raimo/Smallpdf/2.3.0.0/raimo.Smallpdf.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: raimo.Smallpdf
+PackageVersion: 2.3.0.0
+PackageLocale: en-US
+Publisher: raimo
+PackageName: Smallpdf
+License: ???
+ShortDescription: pdf management programm
```
Comment (ItzLevvie) [inline manifests/r/raimo/Smallpdf/2.3.0.0/raimo.Smallpdf.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Smallpdf is the first and only PDF software you’ll love. We have all the tools you’ll need to start, manage, and finish your work with digital documents.
```

File: manifests/r/raimo/Smallpdf/2.3.0.0/raimo.Smallpdf.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: raimo.Smallpdf
+PackageVersion: 2.3.0.0
+PackageLocale: en-US
+Publisher: raimo
+PackageName: Smallpdf
+License: ???
```
Comment (ItzLevvie) [inline manifests/r/raimo/Smallpdf/2.3.0.0/raimo.Smallpdf.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**